vg/runtime/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:14.04
+
+MAINTAINER Charles Markello, cmarkell@ucsc.edu
+
+RUN mkdir /opt/vg/
+
+RUN pwd
+RUN ls -l
```

Comment:
I'd probably remove these two. Unless you have a good reason for leaving them in. I see how they might help during initial development but I don't see their value at this point.